### DISCUSSION TITLE
Add Fonts

### DISCUSSION THREAD

**Discussion by greenieves** [notaapp/talk#286] [Feature Requests]
Please consider adding system fonts or at least just palantino to the list of fonts available.

**Comment by vladstudio** [comment]
Hey, I think this is alreayd supported. I was able to set the editor font to a custom one I had installed:

<img width="398" alt="image" src="https://user-images.githubusercontent.com/914320/183308735-fdb38deb-7346-413c-bd08-336e6b0c083b.png">

**Reply by greenieves** [reply]
Oh that's great. Could you share how you installed the custom font?

**Reply by vladstudio** [reply]
Hmm, the standard way - downloaded some font files, clicked them to open Font Book app, clicked "Install" button.

**Comment by astoilkov** [comment]
I will research a little for security concerns. Other than that we can easily include fonts from `/System/Library/Fonts`.
